src/librustc_error_codes/error_codes/E0009.md

```diff
@@ -6,7 +8,9 @@ This limitation may be removed in a future version of Rust.
 
 Erroneous code example:
 
-```compile_fail,E0009
+```
+#![feature(move_ref_pattern)]
+
```

Comment:
Shouldn't we let it fail without the feature enabled and extend the documentation to explain that the nightly-only `#![feature(move_ref_pattern)]` allows this to be valid (until this gets promoted to stable, then we can update the documentation)?

Reply:
See below.